engine/crates/composition/src/subgraphs/enums.rs

```diff
@@ -3,12 +3,24 @@ use super::*;
 #[derive(Default)]
 pub(super) struct Enums {
     values: BTreeSet<(DefinitionId, StringId)>,
+    deprecated: HashMap<(StringId, StringId), Option<StringId>>,
```

Comment:
not super obvious how `(StringId, StringId)` is a path for an enum

Reply:
True. Will document.